src/server/req_holdjob.c

```diff
@@ -141,9 +142,9 @@ req_holdjob(struct batch_request *preq)
 	time_t           now;
 
 
-	pjob = chk_job_request(preq->rq_ind.rq_hold.rq_orig.rq_objname, preq, &jt);
-	if (pjob == NULL)
+	if (FALSE == chk_job_request(preq->rq_ind.rq_hold.rq_orig.rq_objname, preq, &jt, &js, &pjob, NULL, NULL))
```

Comment:
Align the second line under if condition.

Reply:
Fixed in rc 17 https://github.com/PBSPro/pbspro/pull/590/commits/f94a3405ef13edaa87df40a89af36420b9627c7b